### PR TITLE
Enhancement/107 subclassof support

### LINKED ISSUE #107: Support rdfs:subClassOf
As a user, I want to be able to view and specify supertypes of a concept (based on the `rdfs:subClassOf` property).

Consider designing a configurable generic solution connected to `skos:broader`-based parent terms as there may be several properties used to build term hierarchies.

This is frontend for https://github.com/opendata-mvcr/termit/issues/85

### PR DESCRIPTION
**Pull request by ledsoft**
Resolves #107

### CODE REVIEW

**Comment by ledsoft** [comment]
[skos:broader se neinferuje z rdfs:subClassOf](https://trello.com/c/APH29Ij8/356-skosbroader-se-neinferuje-z-rdfssubclassof)

File: src/component/term/ParentTermSelector.tsx [hunk 1]
```diff
@@ -116,23 +131,58 @@ export class ParentTermSelector extends React.Component<
...
   };
 
+  private valueToTermUpdate(value: Term[]) {
+    const update: Partial<Term> = {};
+    const term = this.props.term;
+    const valueIris = value.map((t) => t.iri);
+    TERM_BROADER_SUBPROPERTIES.forEach((sp) => {
+      update[sp.attribute] = Utils.sanitizeArray(term[sp.attribute]).filter(
+        (t) => valueIris.indexOf(t.iri) !== -1
+      );
```
Comment (psiotwo) [inline src/component/term/ParentTermSelector.tsx:168]: repeated code

**Review by psiotwo — DISMISSED** [review]
lgtm, not checked the functionality

**Review by psiotwo — APPROVED** [review]
lgtm, not tested the functionality

**Pull request merged**